Behavioral.Automation.Playwright/Behavioral.Automation.Playwright/Pages/MainPage.cs

```diff
@@ -1,12 +1,15 @@
 using Behavioral.Automation.Configs;
 using Behavioral.Automation.Playwright.Configs;
+using Behavioral.Automation.Playwright.Selectors;
 using Behavioral.Automation.Playwright.Services.ElementSelectors;
 
 namespace Behavioral.Automation.Playwright.Pages;
 
 class MainPageExample : ISelectorStorage
 {
     private static readonly string Id = ConfigManager.GetConfig<Config>().SearchAttribute;
+    
+    public ButtonSelector IncrementCountButton = new() {XpathSelector = "//button[@data-automation-id='increment-count-button']"};
```

Comment:
I always wanted to know, why we have IdSelector and XpathSelector, for main selector, while secondary selectors are always xpath ? Isn't it little bit confusing ?

What I mean is following - 
E.G. I have DropdownSelector class. It extends ElementSelector, so gets IdSelector and XpathSelector.
In DropdownSelector itself, we have ItemSelector, MenuSelector. Those two should always be written by Xpath, cause there is no IdSelector side for them. So what's the point of having it for main selector ?

Reply:
Discussed internally.
The idea was to define elements like dropdowns using the following code:

    public DropdownSelector ExonJunctionSpanDropdown = new()
    {
        Selector = new ElementSelector {XpathSelector = "//select[@name='PRIMER_ON_SPLICE_SITE']"},
        MenuSelector = new ElementSelector {XpathSelector = "//select[@name='PRIMER_ON_SPLICE_SITE']"},
        ItemSelector = new ElementSelector {XpathSelector = "//option"},
        ItemSelectionSelector = new ElementSelector {XpathSelector = "//option"}
    };